cal/schema.py

```diff
@@ -1,5 +1,6 @@
 from flask.ext.sqlalchemy import SQLAlchemy
 import flask.ext.whooshalchemy as whooshalchemy
+from pytz import timezone
```

Comment:
Right now, the database can't handle dealing with naive/aware datetimes, so I have to manually strip and re-add the timezones. I'm going to look into alternatives after this.